lib/auth/methods.go

```diff
@@ -438,6 +438,8 @@ func (s *Server) authenticateHeadless(ctx context.Context, req AuthenticateUserR
 		return nil, trace.Wrap(err)
 	}
 
+	emitHeadlessLoginEvent(ctx, events.UserHeadlessLoginRequestedCode, s.emitter, ha, nil)
```

Comment:
@Joerger I've added events as per RFD105, do you think it's ok to log this one? I think it's helpful, but not sure about the DoS concerns you discussed in the RFD.

Reply:
I think it is ok, the endpoint is rate limited, and it's better to log if someone is still trying to DOS.